dvf.api.php

```diff
@@ -7,55 +7,40 @@
  * @ingroup svf
  */
 
-use Drupal\dvf\Plugin\VisualisationStyleInterface;
+use Drupal\dvf\Plugin\VisualisationInterface;
 
 /**
  * @addtogroup hooks
  * @{
  */
 
 /**
- * Alter the configuration settings prior to building visualisation.
- *
- * @param array $configuration
- *   An array of configuration options.
- * @param \Drupal\dvf\Plugin\VisualisationStyleInterface $visualisation_style
- *   Visualisation style instance.
- */
-function hook_dvf_style_configuration_alter(array &$configuration, VisualisationStyleInterface $visualisation_style) {
```

Comment:
@jez500 I have removed it because it was causing issues and invoking an `alter` hook a dozens of time per visualisation. I will see if I can come up with another solution.